mava/environment_loop.py

```diff
@@ -338,6 +348,36 @@ def _compute_episode_statistics(
     ) -> None:
         pass
 
+    def get_counts(self) -> Any:
+        if hasattr(self._executor, "_counts"):
```

Comment:
I didn't see this being used anywhere? I.e. executor examples that have this attribute. What is the use case for this?

Reply:
So this is for systems that haven't been moved to mava scaling yet. These systems use `_counts` .